src/Illuminate/Collections/Arr.php

```diff
@@ -702,11 +702,12 @@ public static function pull(&$array, $key, $default = null)
      * Convert the array into a query string.
      *
      * @param  array  $array
+     * @param  int-mask $encoding_type (optional) PHP Query encoding type.
```

Comment:
Since we can narrow it down, we should do just that, to improve IDE support:
```suggestion
     * @param  int-mask-of<PHP_QUERY_*> $encoding_type (optional) PHP Query encoding type.
```